src/main/java/sirius/biz/storage/layer2/ProcessBlobChangesLoop.java

```diff
@@ -91,8 +105,38 @@ protected void deletePhysicalObject(@Nonnull Blob blob) {
         }
     }
 
-    protected void invokeCreatedOrRenamedHandlers(Blob blob) {
-        createdOrRenamedHandlers.forEach(handler -> {
+    protected void invokeCreatedHandlers(Blob blob) {
+        createdHandlers.forEach(handler -> {
+            try {
+                handler.execute(blob);
+            } catch (Exception e) {
+                buildStorageException(e).withSystemErrorMessage(
+                        "Layer 2: %s failed to process the new blob %s (%s) in %s: (%s)",
+                        handler.getClass().getSimpleName(),
+                        blob.getBlobKey(),
+                        blob.getFilename(),
+                        blob.getSpaceName()).handle();
+            }
+        });
+    }
```

Comment:
this same method is repeaded 3 times - pass list of handlers and name for log msg into a common method?